owners/package.json

```diff
@@ -17,7 +17,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "lodash": "4.17.11",
-    "probot": "7.5.0",
+    "probot": "9.3.1",
```

Comment:
I'm not sure if this major version upgrade might introduce breaking changes, so unless @erwinmombay knows I'm holding off approving this until I can fire up the app myself.

Reply:
Oh just now seeing the whole conversation above, seems to be in agreement :sweat_smile: